src/commands/notify.yml

```diff
@@ -40,27 +40,30 @@ parameters:
     type: string
     default: coda.io
   circle_token:
-    default: ${CIRCLECI_TOKEN}
+    default: $CIRCLECI_TOKEN
     description: |
       Env var of a token granted read access to the CircleCI api
     type: string
 steps:
+- run:
+    name: Set variables
+    command: |
+      echo "export CODA_API_TOKEN=<<parameters.coda_api_token>>" >> "$BASH_ENV"
```

Comment:
Nit: for consistency, change all the params to be lowercased?  ex: EMAIL_DOMAIN -> email_domain, etc.